### PR TITLE
Adds a basic function check for GetStats

### PR DESCRIPTION
**Pull request by alvestrand**
This is a skeleton that can be extended for any number of tests.
It passes on Chrome Canary with the "experimental features" flag set.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/4499)

### CODE REVIEW

**Comment by alvestrand** [comment]
@henbos please take a look - it seems like an easy way to check on Firefox compatibility.

File: webrtc/getstats.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      for (let stats of report.values()) {
+        reportDictionary[stats.id] = stats;
+      }
+      showStats.innerHTML = JSON.stringify(reportDictionary, null, 2);
+      // Check the stats properties.
+      assert_not_equals(report, null);
+      let sessionStat = getStatsRecordByType(report, 'peer-connection');
+      assert_not_equals(sessionStat, null, 'Did not find peer-connection stats');
+      assert_exists(sessionStat, 'dataChannelsOpened');
+      assert_equals(sessionStat.dataChannelsOpened, 1);
```
Comment (henbos) [inline webrtc/getstats.html]: nit: Wouldn't it make more sense to call sessionStats peerConnectionStats?

File: webrtc/getstats.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  }
+
+  var onIceConnectionStateChange = test.step_func(function(event) {
+    // Wait until connection is established.
+    // Note - not all browsers reach 'completed' state, so we're
+    // checking for 'connected' state instead.
+    if (gFirstConnection.iceConnectionState != 'connected') {
+      return;
+    }
+    gFirstConnection.getStats()
```
Comment (henbos) [inline webrtc/getstats.html]: A follow-up CL should add one using the selector argument, though that would require tracks.

File: webrtc/getstats.html [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+      let reportDictionary = {};
+      for (let stats of report.values()) {
+        reportDictionary[stats.id] = stats;
+      }
+      showStats.innerHTML = JSON.stringify(reportDictionary, null, 2);
+      // Check the stats properties.
+      assert_not_equals(report, null);
+      let sessionStat = getStatsRecordByType(report, 'peer-connection');
+      assert_not_equals(sessionStat, null, 'Did not find peer-connection stats');
+      assert_exists(sessionStat, 'dataChannelsOpened');
```
Comment (henbos) [inline webrtc/getstats.html]: (Sweet, this does not have to be updated even if https://github.com/w3c/webrtc-stats/pull/125 lands.)

**Review by henbos — APPROVED** [review]
Looks good to me.

**Pull request merged**